text/0000-official-utility-buildpacks.md

```diff
@@ -46,6 +46,12 @@ Examples of potential utility buildpacks include:
 
 The utility buildpacks will be owned, supported, and maintained by the [Implementation team](https://github.com/buildpacks/community/blob/main/TEAMS.md#implementation-team).
 
+The criteria for accepting a utility buildpack into the project is as follows:
+- The behavior provided by the buildpack MUST be defined in the spec or an extension spec.
+- The buildpack MUST support arbitrary stacks
+- The buildpack MUST support the latest API version
```

Comment:
Do we care about it support multiple APIs or only one?

Reply:
I don't think we should support multiple APIs. We should support the latest and folks can use older versions if they require an older APIs, or update the lifecycle :)

Reply:
Are we going to bugfix or maintain multiple versions then for older Buildpack APIs?

Reply:
Maybe we should have the same policy as we would for libcnb